core/src/main/java/io/opentelemetry/android/instrumentation/AndroidInstrumentationServices.kt

```diff
@@ -24,26 +26,25 @@ interface AndroidInstrumentationRegistry {
      */
     fun getAll(): Collection<AndroidInstrumentation>
 
-    /**
-     * Stores an instrumentation as long as there is not other instrumentation already registered with the same
-     * type.
-     *
-     * @param instrumentation The instrumentation to register.
-     * @throws IllegalStateException If the instrumentation couldn't be registered.
-     */
-    fun register(instrumentation: AndroidInstrumentation)
-
     companion object {
-        private var instance: AndroidInstrumentationRegistry? = null
+        private var instance: AndroidInstrumentationServices? = null
 
         @JvmStatic
-        fun get(): AndroidInstrumentationRegistry {
+        fun get(): AndroidInstrumentationServices {
             if (instance == null) {
-                instance = AndroidInstrumentationRegistryImpl()
+                instance = AndroidInstrumentationServicesImpl()
             }
             return instance!!
         }
 
+        /**
+         * Convenience method for [AndroidInstrumentationServices.getByType].
+         */
+        @JvmStatic
+        fun <T : AndroidInstrumentation> getService(type: Class<out T>): T? {
```

Comment:
Same concerns here about `Service`. Would prefer to call this `getInstrumentation()`.

Reply:
Thanks, it's updated now.